.github/workflows/build_and_test.yml

```diff
@@ -484,6 +484,7 @@ jobs:
         apt-get install -y libcurl4-openssl-dev libgit2-dev libssl-dev libxml2-dev
         Rscript -e "install.packages(c('devtools'), repos='https://cloud.r-project.org/')"
         Rscript -e "devtools::install_version('lintr', version='2.0.1', repos='https://cloud.r-project.org')"
+        Rscript -e "install.packages(c('ggplot2'), repos='https://cloud.r-project.org/')"
```

Comment:
A very elementary question: what is the difference between adding it here and adding it to line 509? 

Add it to line 509 will report an error: `there is no package called ‘mvtnorm’`, then `there is no package called ‘statmod’`

Reply:
It seems that I didn't check those before.
```
$ git grep mvtnorm
R/pkg/R/mllib_clustering.R:#' library(mvtnorm)
R/pkg/tests/fulltests/test_mllib_clustering.R:  #' library(mvtnorm)
mllib/src/test/scala/org/apache/spark/ml/clustering/GaussianMixtureSuite.scala:      library(mvtnorm)
```